components/br-stream/src/endpoint.rs

```diff
@@ -167,11 +168,18 @@ where
         if let Some(cli) = self.meta_client.as_ref() {
             let cli = cli.clone();
             let range_router = self.range_router.clone();
+
+            info!("register backup stream task {:?}", task);
+
             self.pool.block_on(async move {
                 let task_name = task.info.get_name();
                 match cli.ranges_of_task(task_name).await {
                     Ok(ranges) => {
-                        debug!("backup stream register ranges to observer");
+                        info!(
+                            "backup stream {:?} ranges len {:?}",
+                            task,
+                            ranges.inner.len()
+                        );
```

Comment:
use structured logging for this and line 172 (something like `info!("backup stream"; "task" => ?task)`